README.md

```diff
@@ -1,16 +1,67 @@
-# Title
+# Building Cells: Getter and Setter Methods
```

Comment:
Okay..

It looks like MDN treats `get` like the standard thing to use whenever you've got a method that returns calculated or dynamic data, so we probably should too.

> Sometimes it is desirable to allow access to a property that returns a dynamically computed value, or you may want to reflect the status of an internal variable without requiring the use of explicit method calls. In JavaScript, this can be accomplished with the use of a getter.

I think the shape example might still work - you can use the `get` version of `area` to calculate based on radius _or_ you can use the `set` version of
`area` to _set_ the radius based on the area:

```
get area() {
return Math.pi * this.radius * this.radius
}

set area(area) {
this.radius = Math.sqrt(area / Math.pi)
}
```
We can ignore `defineProperty` and stick to just `get` and `set` inside classes, similar to what sitepoint has here in its examples: https://www.sitepoint.com/javascript-private-class-fields/

Reply:
I like this links explanation of the two: https://javascriptplayground.com/es5-getters-setters/